modules/sandboxed-containers-selecting-nodes.adoc

```diff
@@ -0,0 +1,7 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/deploying_sandboxed_containers.adoc
+
+[id="sandboxed-containers-selecting-nodes_{context}"]
+
+= Selecting nodes for OpenShift sandboxed containers
```

Comment:
You can use your `OpenShift sandboxed containers` attribute here

Reply:
PV2/FCC Compliance - not to use attributes in the title